src/return-visitor.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @desc returnVisitor() Fires code when a user returns to an experiment
+ *
+ * @param {Function} callback - The code to execute if the user fits the criteria
+ * @param {int} cutoff - Number of seconds since last visit to consider it a return visit
+ * @param {bool} [optFirstTime=true] - Whether or not to run the code on the first visit
+ *
+ * @return {bool} - Whether the code was run or not
+ */
+
+import cookie from './cookie';
+
+function returnVisitor(callback, cutoff, optFirstTime = true) {
```

Comment:
Can we make callback optional? I could see using this a lot in audiences where you don't need to call something but just return true or false